src/python/turicreate/toolkits/audio_analysis/audio_analysis.py

```diff
@@ -65,10 +65,13 @@ def load_audio(path, with_path=True, recursive=True, ignore_failure=True, random
     if _fnmatch(path, '*.wav'):    # single file
         all_wav_files.append(path)
     elif recursive:
-        for (dir_path, _, file_names) in _os.walk(path):
```

Comment:
This will only fail if the user gives a directory that doesn't exist. It should also fail if the user gives a single file which does not exist.

It's probably best to check early (i.e. outside of the `if`/`elif`) if the path exists.